OCP/AdaptGuess_UserInput.m

```diff
@@ -28,7 +28,10 @@
 % Last edit date: 
 % --------------------------------------------------------------------------
 
-guess.a(guess.a < S.bounds.a.lower) = S.bounds.a.lower;
+for i=1:length(bounds.a.lower)
```

Comment:
For my understanding, why only do this for the activations? Wouldn't it be possible that your IG is also out of bounds when basing the bounds on a suggested Default_Coordinate_Bounds.csv file and not the IG itself like before?